### PR TITLE
docs: updated project names to aws lambda power tools for typescript

### PR DESCRIPTION
**Pull request by goverdhan07**
## Description of your changes

 - updated project names to `AWS Lambda Power tools for TypeScript`

### Related issues, RFCs
#624 

[#624 ](https://github.com/awslabs/aws-lambda-powertools-typescript/issues/624)  
[#ZZZZZ](https://github.com/awslabs/aws-lambda-powertools-typescript/issues/ZZZZZ)

### PR status

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

## Checklist

- [x] [My changes meet the tenets criteria](https://awslabs.github.io/aws-lambda-powertools-python/#tenets)
- [x ] I have performed a *self-review* of my own code
- [x ] I have *commented* my code where necessary, particularly in areas that should be flagged with a TODO, or hard-to-understand areas
- [x ] I have made corresponding changes to the *documentation*
- [x ] I have made corresponding changes to the *examples*
- [x ] My changes generate *no new warnings*
- [x ] The *code coverage* hasn't decreased
- [ ] I have *added tests* that prove my change is effective and works
- [x ] New and existing *unit tests pass* locally and in Github Actions
- [x ] Any *dependent changes have been merged and published* in downstream module
- [x ] The PR title follows the [conventional commit semantics](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/main/.github/semantic.yml#L2)

### Breaking change checklist

- [ ] I have documented the migration process
- [ ] I have added, implemented necessary warnings (if it can live side by side)

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by dreamorosi** [comment]
Hi @goverdhan07 thanks for the PR, will review it early next week.

We really appreciate you taking the time to open a PR, next time could you please leave a comment under the issue you've referenced before start working, like asked in the Contributing guidelines?

**Review by dreamorosi — CHANGES_REQUESTED** [review]
Thank you again for the PR, it's already looking good, just one minor note: 

The various README.md files (both in the proejct root & in the packages roots i.e. `packages/logger/README.md`) also mention the existence of Powertools for other runtimes - [example](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/89274b4a54bdd811deba4fd6edc53031a19b85e5/README.md?plain=1#L7). Could you please apply the same naming to that line - i.e. AWS Lambda Powertools for Python, etc.?

I'm going to request a review from a second maintainer so that when we have 2 approvals we can merge it.

**Note for other reviewers**: There is an occurrence of the name that appears mid document [here](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/89274b4a54bdd811deba4fd6edc53031a19b85e5/CONTRIBUTING.md?plain=1#L148) and that hasn't been updated, I think it should stay as it is as it falls under the usage described [here](https://github.com/awslabs/aws-lambda-powertools-roadmap/issues/80#issuecomment-1060359500).

**Review by flochaz — APPROVED** [review]

**Review by dreamorosi — DISMISSED** [review]
Looks good to me, thanks for the PR!

File: packages/commons/README.md
Comment (dreamorosi) [inline packages/commons/README.md:26]:
```suggestion
The AWS Lambda Powertools for TypeScript utilities (which from here will be referred as Powertools) follow a modular approach, similar to the official [AWS SDK v3 for JavaScript](https://github.com/aws/aws-sdk-js-v3).
```

**Review by dreamorosi — APPROVED** [review]

File: packages/metrics/README.md
Comment (saragerion) [inline packages/metrics/README.md:26]:
```suggestion
The AWS Lambda Powertools for TypeScript utilities follow a modular approach, similar to the official [AWS SDK v3 for JavaScript](https://github.com/aws/aws-sdk-js-v3).  
```

**Review by saragerion — COMMENTED** [review]
Thanks for this PR :)

Minor change needed, otherwise looks good.

File: packages/metrics/README.md
Comment (goverdhan07) [reply packages/metrics/README.md]: updated :)

**Review by saragerion — APPROVED** [review]
LGTM!

**Review by dreamorosi — APPROVED** [review]

**Pull request merged**
